### PR TITLE
Add more telecomms variations

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

for mappers

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

 no fun

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Test failure real.

**Comment by chromiumboy** [comment]
What does this PR do? The title and description aren't clear

**Comment by lzk228** [comment]
> What does this PR do? The title and description aren't clear

bruh, just check the code part

File: Resources/migration.yml [hunk 1]
```diff
@@ -276,6 +276,6 @@ ClothingNeckFlowerWreath: ClothingHeadHatFlowerWreath
 ClothingHeadHatFlowerCrown: ClothingHeadHatFlowerWreath
 BriefcaseSyndieBase: null
 
-# 2024-04-08
+# 2024 04-08
```
Comment (QuietlyWhisper) [inline Resources/migration.yml:279]: ?

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**